iOS Example Frame SPM/iOS Example Frame SPM.xcodeproj/project.pbxproj

```diff
@@ -376,15 +409,38 @@
 			children = (
 				CBA81CAA27ABDF21000DF192 /* iOS Example Frame */,
 				CB9412D82825714C00C889E8 /* iOS Example FrameUITests */,
+				16857B722A65F15F005CAE39 /* iOS Example Frame Integration Tests */,
```

Comment:
I believe FrameUITests is running a basic UI test. 

The Integration test should be renamed to Regression Pack as we aim to run each and every scenario around token creation, validation & error checks. 

@okhan-okbay-cko please confirm.

Reply:
yes, @deepesh-vasthimal-cko is right. `iOS Example FrameUITests` was already existent and was having 2 basic test cases, which will continue to run on each PR check as a lightweight+quick confirmation of the code. 

`iOS Example Frame Integration Tests` is the new automated regression tests that's the motive of this PR. It'll include all the detailed test cases. Until we use GHA, it'll be run by hand. After that, we can optimise it to be required to be run before each and every PR merge on-demand/ manually.

For the naming, it can be renamed yeah. Regression testing denotes the purpose of this better 👍

Reply:
Renamed it